### ISSUE TITLE
New Feature Discussion: using a persistent config file

### ISSUE THREAD

**Issue by CaseyMacPherson** [chris-peterson/pwsh-gitlab#11]
What is the feeling about introducing a persistent configuration file for individuals that have multiple gitlab systems.

possibly using `~/.gitlabcli/config.yml`

Associated commands that should be created to facilitate
* `Add-GitlabSystem -Name "gitlab.mygitlabinstance" -ApiToken "8fhs63"`
* `Remove-GitlabSystem -Name "gitlab.mygitlabinstance"`
* `Set-GitlabSystem -Name "gitlab.mygitlabinstance"`

The one concern is that this requires an external package `powershell-yaml`. If that's a concern, then `JSON` format will work as well. I've included both examples below

## Example configurations
**YAML**
```yml
ActiveSite: gitlab.com
gitlab.com:
  - ApiToken: xyz12345
gitlab.mygitlabinstance:
  - ApiToken: "8fhs63"
```
**JSON**
```json
{
  "ActiveSite": "gitlab.com",
  "gitlab.com": {
    "ApiToken": "xyz12345"
  },
  "gitlab.mygitlabinstance": {
    "ApiToken": "8fhs63"
  }
}
```

## Cli Output 
**YAML**
```pwsh
> $config=Get-Content ./config.yml | ConvertFrom-Yaml
> $config.ActiveSite
gitlab.com

> $config[$config.ActiveSite].ApiToken
8fhs63
```
**JSON**
```pwsh
$config = Get-Content ./config.json | ConvertFrom-Json -AsHashTable
> $config.ActiveSite
gitlab.com

> $config[$config.ActiveSite].ApiToken
8fhs63
```

**Comment by chris-peterson** [comment]
Right now, I have a personal machine that I use to dev against gitlab.com and a work machine that I use to dev against a private installation.  But it would be nice to be able to switch profiles rather than machines :)

**Comment by CaseyMacPherson** [comment]
After a little more consideration I'd like to change 2 items

1.  config key `ActiveSite`  to `DefaultSite` 
1. `Set-GitlabSystem`  to `Set-GitlabDefaultSystem`

**Comment by chris-peterson** [comment]
Resolved by #13
